spec/rubocop/cop/style/it_block_parameter_spec.rb

```diff
@@ -0,0 +1,218 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::ItBlockParameter, :config do
+  context '>= Ruby 3.4', :ruby34, unsupported_on: :parser do
+    context 'EnforcedStyle: allow_named_parameter' do
+      let(:cop_config) { { 'EnforcedStyle' => 'allow_named_parameter' } }
+
+      it 'registers an offense when using a single numbered parameters' do
+        expect_offense(<<~RUBY)
+          block { do_something(_1) }
+                               ^^ Use `it` block parameter.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          block { do_something(it) }
+        RUBY
+      end
+
+      it 'registers an offense when using twice a single numbered parameters' do
```

Comment:
These are two offenses that can be autocorrected one by one with Ruby LSP for example. If you correct only once, it'll be a syntax error because `it`/`_1` is mixed.

It's shown better the way you currently have it but maybe it could be done in a different way?

Reply:
Interesting feedback. As for `textDocument/codeAction`, since it's uncertain whether this feature alone will cause issues, it might be worth considering separately.

Reply:
I don't believe this is the only cop where this is the case. You can discard this since no one has  yet reported it as an issue for the others